docs/install_guides/pypi-openvino-dev.md

```diff
@@ -78,20 +84,32 @@ python -m pip install --upgrade pip
 
 ### Step 4. Install the Package
 
-Run the command below: <br>
+To install and configure components of the development package for working with specific frameworks, use the `pip install openvino-dev[extras]` command, where `extras` is a list of extras from the table below: 
```

Comment:
To install and configure components ---> To install and configure the components